storage/scripts/poc.sh

```diff
@@ -1,26 +1,24 @@
 #!/bin/bash
-#
+# SPDX-License-Identifier: Apache-2.0
 # Copyright (c) 2022 Intel Corporation
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
+# Copyright (c) 2022 Dell Inc, or its subsidiaries.
+
+set -euxo pipefail
+
+# DOCKER_COMPOSE setup
+DC=docker-compose
+
+if [ "$(command -v $DC)" == "" ]
+then
+    DC="docker compose"
+fi
 
 usage() {
-	echo "Usage: poc.sh [start|stop|tests|logs]"
+    echo "Usage: poc.sh [start|stop|tests|logs]"
 }
 
 tests_poc() {
-	docker-compose ps -a
+    $DC ps -a
```

Comment:
Do we need "bash -c ${DC} ps -a" here or no?

Reply:
i don;t know why you did  `bash -c ${DC}` everywhere.. it works without it...